app/parse.py

```diff
@@ -8,5 +13,18 @@
     duration: str
 
 
+def parse_single_course(course_soup: Tag) -> Course:
+    name = course_soup.select_one("a.ProfessionCard_title__Zq5ZY > h3").text
+    short_description = course_soup.select_one(
+        ".typography_landingTextMain__Rc8BD.mb-32").text
```

Comment:
The CSS selector `.typography_landingTextMain__Rc8BD.mb-32` might be too specific and could break if the class names change. Ensure this selector matches the current webpage structure.